scripts/build_npm.ts

```diff
@@ -102,8 +101,9 @@ await new Deno.Command("npm", {
     stdout: "inherit",
     stderr: "inherit",
 }).output();
+
 await new Deno.Command("npm", {
-    args: ["exec", "typedoc"],
+    args: ["exec", "typedoc", "--", "--entryPointStrategy", "expand", "./src"],
```

Comment:
I don't know if this gives you what you want, but at least it is no longer crashing